_docs/install-and-upgrade.md

```diff
@@ -155,19 +168,18 @@ may be unstable or insecure. This is not production-ready, so use at your own
 risk!<br>
 It is also not recommended to run this as a `root` user.
 
-First, make sure [Node.js](https://nodejs.org/) v4 or more recent is installed
-with `node --version`, as well as. If not, follow the instructions given on the
+First, make sure [Node.js](https://nodejs.org/) v6 or more recent is installed
+with `node --version`, as well as [yarn](https://yarnpkg.com). If not, follow the instructions given on the
 [official documentation](https://nodejs.org/en/download/package-manager/) by
 choosing your distribution in the list.
 
 The following commands install the development version of The Lounge:
 
 ```sh
-git clone https://github.com/thelounge/lounge
-cd lounge
-npm install
-NODE_ENV=production npm run build
-npm link
+git clone https://github.com/thelounge/thelounge
+cd thelounge
+yarn install
+NODE_ENV=production yarn build
```

Comment:
Uh oh, there is no global version of `yarn link`? That's a shame, I'd rather keep `thelounge start` rather than `node index.js start`...
Maybe keep `npm link` there until yarn has a global version or something?

Reply:
Turns out there is a `yarn link`. I'll add that back in.

Reply:
There is, but I don't know if it actually makes the binary available as `thelounge` globally.

Reply:
Wait, nevermind, `yarn link` does exactly what we want it to do 👏